<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1306</width>
    <height>802</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1281</width>
      <height>741</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="leftLA">
       <item>
        <widget class="ScorePropertyWidget" name="bodyBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="mindBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="soulBox" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="nrBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="srBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="speedBox" native="true"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="midLA">
       <item>
        <widget class="PoolPropertyWidget" name="hpBox" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item>
        <widget class="PoolPropertyWidget" name="wpBox" native="true"/>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTabWidget" name="rcmLA">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="raceTab">
          <attribute name="title">
           <string>Set race</string>
          </attribute>
          <widget class="QComboBox" name="raceCombo">
           <property name="geometry">
            <rect>
             <x>18</x>
             <y>60</y>
             <width>381</width>
             <height>22</height>
            </rect>
           </property>
          </widget>
          <widget class="QPushButton" name="raceSet">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>110</y>
             <width>75</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>SELECT</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="classTab">
          <attribute name="title">
           <string>Class</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="skillLA">
       <item>
        <widget class="ScorePropertyWidget" name="strengthBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="nimbleBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="perceptionBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="diplomacyBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="magicBox" native="true"/>
       </item>
       <item>
        <widget class="ScorePropertyWidget" name="connectionBox" native="true"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1306</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScorePropertyWidget</class>
   <extends>QWidget</extends>
   <header location="global">scorepropertywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PoolPropertyWidget</class>
   <extends>QWidget</extends>
   <header location="global">poolpropertywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
